src/hydra/parseHydraDocumentation.test.js

```diff
@@ -1365,7 +1365,7 @@ test("Invalid docs JSON", async () => {
 });
 
 test("Invalid entrypoint JSON", async () => {
-  let entrypoint = `{foo,}`;
+  const entrypoint = `{foo,}`;
```

Comment:
you don't need brackets, use a simple quote (`'`) instead :)

Reply:
This change came from #34. It has been removed.